### PR TITLE
Youtube Clone - Basic HTML page Structure

### PR DESCRIPTION
**Pull request by mke2111**
**The Youtube-clone Homepage**

_We worked with the new feature branch "trials"  and designed a Youtube "look-alike" Homepage using HTML and CSS_

### CODE REVIEW

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
## Hello @mke2111 and @che30 

### First of all, congrats in working on your first Microverse project 🌟

Add a more descriptive pull request mentioning what you were able to do in this project [check this guide](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/). The title should reflect the feature you added.

This Code Review Request will be marked as invalid on your Dashboard, so please submit a new one once you are ready!

Happy coding!
By Uduak Essien
Slack @acushla

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (krys2fa) [inline .vscode/settings.json]: - [ ] [OPTIONAL] Files not related to the project should not be added to your commits. 
You can include this folder in the .gitignore in the future.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+                <div class="details">
+                    <div class="title">
+                        <h2>Master KG - Jerusalema [Feat. Nomcebo] (Official Music Video)</h2></div>
+                            <div class="users">
+                                <div class="views"><h5>139,649,180 views•Premiered Dec 13, 2019</h5></div>
+                                <div class="video-details">
+                                    <div style="padding-right: 15px;"><i style="padding-right: 6px;" class="fas fa-thumbs-up"></i>2334</div>
+                                    <div style="padding-right: 15px;"><i style="padding-right: 6px;" class="fas fa-thumbs-down"></i>2234</div>
+                                    <div style="padding-right: 15px;"><i class="fas fa-share" style="padding-right: 6px;"></i>SHARE</div>
+                                    <div style="padding-right: 15px;"><i class="fas fa-ellipsis-h"></i></div>
```
Comment (krys2fa) [inline index.html]: - [ ] [REQUIRED] Inline styles should not be used as they fall under [basic bad CSS practices](https://speckyboy.com/good-bad-css-practices/) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright).
Please remove all inline styles from this file.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+                    <button style="height: 25px;" type="submit"><i class="fas fa-search"></i></button>
+                </form>
+                <div class="icons">
+                    <i class="fas icon-pad fa-arrow-up" ></i>
+                    <i class="fas icon-pad fa-video" ></i>
+                    <i class="fas icon-pad fa-user" ></i>
+                </div>
+            </nav>
+        </header>
+        <div class="videos">
```
Comment (krys2fa) [inline index.html:32]: - [ ] [OPTIONAL] You are encouraged to use the `<main>` tag for the dominant content of the `<body>` of the document.

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Nice work team 👏 
Your project displays good use of semantic HTML as well as CSS classes. 💪 
You have a professional looking README file as well. 👍 

Just a few more adjustments to complete the project. They can be seen in the comments below:
### Status: Changes Required ♻️ 

Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

File: index.html (hunk 2 above)
Comment (mke2111) [reply index.html]: Thanks for the review.
Changes have been made as suggested.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+    height: 20px;
+    width: auto;
+}
+.aside-top a{
+float: right;
+text-decoration: none;
+color: grey;
+font-size: 12px;
+font-weight: bold;
+}
```
Comment (teekaytech) [inline style.css]: Kindly indent your CSS rules block.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+    }
+    .info-style{
+        margin: 10px 20px;
+    }
+    .policy{
+    display:flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    margin-left: 40px;
+    }
```
Comment (teekaytech) [inline style.css]: Use proper indentation for these blocks.

**Review by teekaytech — CHANGES_REQUESTED** [review]
# Status: Changes required :recycle: :vertical_traffic_light: 

## Hi team :wave: 

### You've done an awesome job so far :ok_hand:. But there are still some changes that are necessary for this project to be approved. 

**Let us quickly make some finishing touches and request another review.** 

Kindly review my points below :point_down: 

- Your `SUBSCRIBE` button is a screenshot. You're required to build your button with the appropriate HTML elements, and not a screenshot.


b

- There's too much horizontal space in-between the `video section` column and the `recommended videos section` column. Kindly see the difference from the image below and make changes as necessary.

Your page:

y

Actual page: 

a

* * *
**You can reach me on slack (@taofeekolalere)
Happy coding :bowing_man:**

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
### The [clone](https://che30.github.io/YoutubeCloneCR1230/) shows a good understanding and usage of embedding images and videos. :100:
##
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @che30  and @mke2111! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know on slack at any time. I will be happy to help.**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**